src/ringmesh/io/io_geomodel.cpp

```diff
@@ -39,7 +39,7 @@
 #include <deque>
 #include <iomanip>
 
-#include <tinyxml2/tinyxml2.h>
+#include <tinyxml2.h>
```

Comment:
I am sorry I have not followed all the discussion on these things so maybe I am too late, but why this choice of including these files without a prefix.

Reply:
It is because we now use the tinyxml2 configuration. We do not specify anything anymore. So it is their choice.